user/enterprise/platform-tips.md

```diff
@@ -255,52 +255,55 @@ $ replicatedctl app start
 
 ### Automate renewal of your Let's Encrypt SSL Certificate with a cron job
 
-Create a file such as `/home/ubuntu/renew-certs.sh` with the following in it:
+1. Create `/home/ubuntu/renew-certs.sh` containing the following script:
 
-```
-#!/bin/bash
+    ```sh
+    #!/bin/bash
 
-set -u
+    set -u
 
-function usage() {
-  echo
-  echo "Usage: $(basename $0)"
-  echo
-  echo "Simple script to renew certs, should be used via a cron. Assumes that certs are already set up."
-  echo
-  echo "See https://docs.travis-ci.com/user/enterprise/platform-tips/#use-a-lets-encrypt-ssl-certificate for initial setup info."
-  echo
+    function usage() {
+      echo
+      echo "Usage: $(basename $0)"
+      echo
+      echo "Simple script to renew certs, should be used via a cron. Assumes that certs are already set up."
+      echo
+      echo "See https://docs.travis-ci.com/user/enterprise/platform-tips/#use-a-lets-encrypt-ssl-certificate for initial setup info."
+      echo
 
-  exit 1
-}
+      exit 1
+    }
 
-if [[ $# -ne 0 ]]; then
-  usage
-fi
+    if [[ $# -ne 0 ]]; then
+      usage
+    fi
 
-replicatedctl app stop
-sudo certbot renew
-replicatedctl app start
-```
-Set the executable permissions you'll need by doing `chmod +x /home/ubuntu/renew-certs.sh`.
+    replicatedctl app stop
+    sudo certbot renew
+    replicatedctl app start
+    ```
+    {: data-file="/home/ubuntu/renew-certs.sh"}
```

Comment:
Is this right? I don't know exactly what it means, but it seems to render regular text

Reply:
Yes, this is what it looks like: https://deploy-preview-2270--musing-lamport-7fb66c.netlify.com/user/enterprise/platform-tips/#automate-renewal-of-your-lets-encrypt-ssl-certificate-with-a-cron-job

Reply:
Oh nice!